src/tests/SConscript

```diff
@@ -73,6 +73,12 @@ def build_tests(prereqs, env):
     # rpc tests
     SConscript('rpc/SConscript')
 
+    # daos middleware consistency corruption tool for testing
+    libs_client += ['dfs']
+    denv.AppendUnique(RPATH_FULL=['$PREFIX/lib64'])
+    daos_mw = denv.d_program('daos_mw_fi', 'daos_mw_fi.c', LIBS=libs_client, RPATH='$PREFIX/lib64')
```

Comment:
You shouldn't need RPATH= setting here
RPATH_FULL is supposed to add any RPATH

Reply:
it's been a while, but it doesn't seem that i would have added that to start with :-)
i am pretty sure that i added it because it was needed..

Reply:
ah i see what you are saying with RPATH_FULL vs RPATH.. yea im not sure why i had that. i'll test removing the rpath. but should i commit now on the branch or should i fix this on master after the branch lands?

Reply:
I'm fine either way.  If someone makes you update, push it in an update